osm2gtfs/core/osm_connector.py

```diff
@@ -125,59 +126,79 @@ def get_routes(self, refresh=False):
 
         # Build routes from master relations
         for rmid, route_master in route_masters.iteritems():
-            members = OrderedDict()
+            itineraries = OrderedDict()
 
             # Build route variant members
             for member in route_master.members:
 
+                # Create Itinerary objects from member route variants
                 if member.ref in route_variants:
                     rv = route_variants.pop(member.ref)
-                    members[rv.id] = self._build_route_variant(rv, result)
+                    itineraries[rv.id] = self._build_itinerary(rv, result,
+                                                               route_master)
 
-                # Route master member was already used before or is not valid
+                # Route variant was already used or is not valid
                 else:
                     rv = result.get_relations(member.ref)
                     if bool(rv):
                         rv = rv.pop()
-                        sys.stderr.write("Route variant was assigned again:\n")
+                        sys.stderr.write("Itinerary was assigned again:\n")
                         sys.stderr.write(
                             "http://osm.org/relation/" + str(rv.id) + "\n")
```

Comment:
Here and in all the other cases, we should use an static variable to not hardcode the same url over and over again? Also, please use `https`.

Reply:
I changed to `https`. However on using a static variable or a global constant and how to handle those concepts in Python is not completely clear to me. A quick research got me to the point, that we probably will need some kind of `const` class, which will bring a lot of good oportunities. But it seems to be bigger and we need to think about it. What about doing this in a separate issue? So far they have always been hardcoded, so maybe let's change it properly after this going in.

Reply:
I'm okay with this. 👍

Reply:
Sure, #102.